transports/janus_rabbitmq.c

```diff
@@ -472,22 +526,59 @@ int janus_rabbitmq_init(janus_transport_callbacks *callback, const char *config_
 		rmq_client->admin_api_enabled = FALSE;
 		if(rmq_admin_api_enabled) {
 			rmq_client->admin_api_enabled = TRUE;
-			JANUS_LOG(LOG_VERB, "Declaring incoming queue... (%s)\n", to_janus_admin);
-			rmq_client->to_janus_admin_queue = amqp_cstring_bytes(to_janus_admin);
-			amqp_queue_declare(rmq_client->rmq_conn, rmq_client->rmq_channel, rmq_client->to_janus_admin_queue, 0, 0, 0, 0, amqp_empty_table);
-			result = amqp_get_rpc_reply(rmq_client->rmq_conn);
-			if(result.reply_type != AMQP_RESPONSE_NORMAL) {
-				JANUS_LOG(LOG_FATAL, "Can't connect to RabbitMQ server: error declaring queue... %s, %s\n", amqp_error_string2(result.library_error), amqp_method_name(result.reply.id));
-				goto error;
+
+			// Set queue options
+			amqp_boolean_t queue_durable_admin = 0;
+			item = janus_config_get(config, config_admin, janus_config_type_item, "queue_durable_admin");
+			if(item && item->value && janus_is_true(item->value)) {
+				queue_durable_admin = 1;
 			}
-			JANUS_LOG(LOG_VERB, "Declaring outgoing queue... (%s)\n", from_janus_admin);
```

Comment:
same as janus api out queue, I'd leave the definition in case queue_name has not been defined